.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - "stable"
```

Comment:
When everything works now, maybe we can start to add more features in the Travis configurations...
For example, testing not only in Linux but also in Mac and Windows environment.
Also, not only with the stable version of Node but also with latest once, etc.